Probesys/bileto
# imp: Rework the design of the filters

## Related issue(s)



https://github.com/Probesys/bileto/issues/562

## How to test manually



- go to the list of tickets
- check the design of filters has changed and looks better
- try to apply some filters, check they all work correctly

## Checklist
